### ISSUE TITLE
how to use fwdgrad for regression problems besides classification problems?

### ISSUE THREAD

**Issue by xgxg1314** [orobix/fwdgrad#6]
Hi,Thank you for your wonderful framework!
I have a question about regression problems using fwdgrad. In your paper, classification problems are solved with cross-entropy loss and softmax activation, then is it possible to solve regression problems by your fwdgrad?

**Comment by DavideTr8** [comment]
Hi, thanks for your words!
Ideally this framework should work with every function. That is also regression problems should be solvable. Anyway right now _functorch_ (the Pytorch package that allows for the computation of _Jvp_) is still in beta, so not all functions are readily supported (we had to implement our own crossentropy). You might need to reimplement your loss before using the framework.

By the way, the paper was not written by us, this is an unofficial implementation since the official one has not been released yet.

**Comment by xgxg1314** [comment]
> 

Ok, i see, i will try to solve regression problems using your code. Thanks for your repositories anyway!

**Comment by DavideTr8** [comment]
You are welcome! We will update the code as soon as new Pytorch functions will be supported (hopefully in the next release)
